From: Juergen Borleis <jbe@pengutronix.de>
Date: Mon, 23 Nov 2015 11:08:02 +0100
Subject: [PATCH] Skov-DO63x: add device trees for the C revisions of the DO63x
 boards

These board revisions seems similar to the revision B. So it is just a copy
of the B revision content.

Signed-off-by: Juergen Borleis <jbe@pengutronix.de>
---
 arch/arm/boot/dts/imx6dl-skov-dol63x-revC.dts | 23 +++++++++++++++++++
 arch/arm/boot/dts/imx6q-skov-dol63x-revC.dts  | 32 +++++++++++++++++++++++++++
 2 files changed, 55 insertions(+)
 create mode 100644 arch/arm/boot/dts/imx6dl-skov-dol63x-revC.dts
 create mode 100644 arch/arm/boot/dts/imx6q-skov-dol63x-revC.dts

diff --git a/arch/arm/boot/dts/imx6dl-skov-dol63x-revC.dts b/arch/arm/boot/dts/imx6dl-skov-dol63x-revC.dts
new file mode 100644
index 000000000000..6e14835fe3c0
--- /dev/null
+++ b/arch/arm/boot/dts/imx6dl-skov-dol63x-revC.dts
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2014 Rose Technology
+ * Copyright 2012 Freescale Semiconductor, Inc.
+ * Copyright 2011 Linaro Ltd.
+ *
+ * The code contained herein is licensed under the GNU General Public
+ * License. You may obtain a copy of the GNU General Public License
+ * Version 2 or later at the following locations:
+ *
+ * http://www.opensource.org/licenses/gpl-license.html
+ * http://www.gnu.org/copyleft/gpl.html
+ */
+
+/dts-v1/;
+#include <dt-bindings/interrupt-controller/irq.h>
+#include "imx6dl.dtsi"
+#include "imx6qdl-cpu.dtsi"
+#include "imx6qdl-cpuB.dtsi"
+
+/ {
+	model = "Freescale i.MX6 SKOV Solo Board";
+	compatible = "skov,dol63x", "fsl,imx6dl";
+};
diff --git a/arch/arm/boot/dts/imx6q-skov-dol63x-revC.dts b/arch/arm/boot/dts/imx6q-skov-dol63x-revC.dts
new file mode 100644
index 000000000000..ece4aa5acfda
--- /dev/null
+++ b/arch/arm/boot/dts/imx6q-skov-dol63x-revC.dts
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2014 Rose Technology
+ * Copyright 2012 Freescale Semiconductor, Inc.
+ * Copyright 2011 Linaro Ltd.
+ *
+ * The code contained herein is licensed under the GNU General Public
+ * License. You may obtain a copy of the GNU General Public License
+ * Version 2 or later at the following locations:
+ *
+ * http://www.opensource.org/licenses/gpl-license.html
+ * http://www.gnu.org/copyleft/gpl.html
+ */
+
+/dts-v1/;
+#include <dt-bindings/interrupt-controller/irq.h>
+#include "imx6q.dtsi"
+#include "imx6qdl-cpu.dtsi"
+#include "imx6qdl-cpuB.dtsi"
+
+/ {
+	model = "Freescale i.MX6 SKOV Quad Board";
+	compatible = "skov,dol63x", "fsl,imx6q";
+};
+
+&sata {
+	status = "disabled";
+};
+
+&hdmi {
+	status = "okay";
+	ddc-i2c-bus = <&i2c2>;
+};
